app/routers/agenda.py

```diff
@@ -39,6 +40,21 @@ def agenda(
     """Route for the agenda page, using dates range or exact amount of days."""
 
     user_id = 1  # there is no user session yet, so I use user id- 1.
+
+    get_a_week_int = 6
```

Comment:
Move all the logic to another function, preferably in a different file.

Reply:
About the logic here, its doing exactly what the rout does, its part of the function.

Get start end day and return a list for a week.
IT has to be the same function has the rout,  i use all the rout parm to get this and i have to return it has part of the page..
 
It is the same as when you search for date it is the same logic

Waiting for your answer🙏

Reply:
done